controllers/article-controllers.js

```diff
@@ -1,12 +1,19 @@
-const { selectArticleById } = require('../models/article-models');
+const { selectArticles, selectArticleById } = require('../models/article-models');
 
 exports.getArticleById = (req, res, next) => {
   const { article_id } = req.params;
   selectArticleById(article_id)
-    .then((result) => {
-      res.status(200).send(result);
-    })
-    .catch((err) => {
-      next(err);
-    });
-};
\ No newline at end of file
+  .then((result) => {
+    res.status(200).send(result);
+  })
+  .catch((err) => {
+    next(err);
+  });
+};
+
+exports.getArticles = (req,res,next) => {
+  
+  selectArticles().then((result) => {
+    res.status(200).send(result)
+  })
+}
```

Comment:
I think this is a bit of a trend in your controllers (well, I suppose I should say in your tests since they decide what your code must do!) of just sending back the result from the model. The result being that the `res.body` you get back in the tests is itself an array, when really what we want is to wrap the array in an object, so our response is not:
```
[
  { ... },
  { ... }
]
```

but is instead:

```
{
  articles: [
    { ... },
    { ... }
  ]
}
```

I would take the opportunity, once you reach the end of a ticket, to do a bit of a refactor in both the tests and controllers to account for this little change (and it is little!)